src/style.css

```diff
@@ -1,3 +1,80 @@
-body {
+html {
+  --header-color: #fff;
+  --nav-link-color: #000;
+  --nav-link-hover-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+.logo {
+  display: flex;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--header-color);
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: var(--nav-link-color);
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0, 0, 0, 0) !important;
+  white-space: nowrap !important;
+}
```

Comment:
```suggestion

```

Reply:
Hi! I don't understand, why i can not use this block of code))